spec/spec_helper.rb

```diff
@@ -1,5 +1,18 @@
 # frozen_string_literal: true
 
+# By default run SimpleCov, but allow an environment variable to disable.
+unless ENV['NOCOVERAGE']
+  require 'simplecov'
+
+  SimpleCov.start do
+    track_files '**/{lib}/**/*.rb'
+    add_filter "/spec/"
+
+    SimpleCov.minimum_coverage_by_file 45 unless ENV['CIRCLE_JOB']
```

Comment:
I picked 45 simply because that is the lowest coverage of any existing file.
We can move this back up, but I was concerned that it may make individuals comfortable seeing that particular command "fail" for them locally and not think much of it.